manifests/t/TravisLane/SteamArtManager/1.4.5/TravisLane.SteamArtManager.locale.en-US.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TravisLane.SteamArtManager
+PackageVersion: 1.4.5
+PackageLocale: en-US
+Publisher: Travis Lane
+PublisherUrl: https://github.com/Tormak9970
+Author: Tormak9970

```

```suggestion
Author: Travis Lane
```
We can use the author's name here